docs/quickstart/ccd.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Customer Cloud Deployments
+hide_table_of_contents: false
+---
+
+[github]: https://github.com/strmprivacy/data-plane-helm-chart/tree/master
+[oma]: https://backstage.strm.in/docs/default/component/organization-management-and-access/
+[jk]: https://docs.google.com/document/d/1ymb8JPaOflX8GZzICV6WCE_L0GGLRrBAPtk0dgeHJdc/edit
+[telepresence]: https://www.telepresence.io/
+[sinks]: https://docs.strmprivacy.io/docs/latest/quickstart/receiving-s3/#create-sink
+[ovh-ingress]: https://docs.ovh.com/au/en/kubernetes/installing-nginx-ingress/
+[profile]: https://console.strmprivacy.io/upgrading
+[values]: https://console.strmprivacy.io/installation/configuration
+[confluent]: https://docs.confluent.io/platform/current/quickstart/ce-docker-quickstart.html#cp-quickstart-step-1
+[makefile]: https://github.com/strmprivacy/data-plane-helm-chart/blob/master/Makefile
+
+This hands-on sessions shows how to get up-and-running with your Customer Cloud Deployment, and verify its
+functionality.
+
+## Get an account with a 'Self Hosted' subscription
+Your [Strm profile page][profile] should show: "Current subscription: Self hosted". Get in touch with your Strm
+representative if you're on a Free or Business subscription.
+
+Once you have this profile active, you can start.
+
+## Install the following tools
+
+* `make` (optional). The Helm repository contains handy `make` targets such as _install_, _upgrade_, _wipe_. None of
+  these are mandatory, one can also execute the commands in the [`Makefile`][makefile] by hand.
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+* [`helm`](http://helm.sh). This Kubernetes package manager is used for installing (and upgrading) your Strm customer
+  dataplane.
+* [`telepresence`][telepresence]. This tool allows your development computer to become _part of_ the k8s cluster, so
+  that you can directly access k8s services.
+* [`k9s`](https://github.com/derailed/k9s) (optional). This _textual user interface_ offers a very convenient way to interact with
+  kubernetes clusters.
+* [`kubectx and kubens`](https://github.com/ahmetb/kubectx) (optional). Very useful tools to switch the default
+  kubernetes context and namespace.
+
+## Start the installation of the Strm customer data-plane
+
+1. download the credentials file `values.yaml` [here][values] into the `data-plane-helm-chart` directory.
```

Comment:
The link "here" is almost unreadable, partly because it's next to another blue text. I missed the link completely.
Maybe rephrase it to "from the console", to make it stand out a bit more.